manifests/testing/disks-images-provider.yaml.in

```diff
@@ -114,6 +114,26 @@ spec:
               - /ready
             initialDelaySeconds: 10
             periodSeconds: 5
+        - name: loopdev
+          command:
+          - sh
+          - -c
+          - |
+            while true; do
+              for i in $(seq 0 5); do
+                  rm -f /dev/loop$i
+                  mknod /dev/loop$i b 7 $i
+              done
+              sleep 100000000
+            done
+          image: alpine:3.6
```

Comment:
maybe you could reuse the previous image? (`{{.DockerPrefix}}/disks-images-provider:{{.DockerTag}}`) since it's already on the system...

Reply:
Sure I can try this.